apps/web/components/result/Result.test.tsx

```diff
@@ -2,10 +2,10 @@
 import { describe, expect, it } from "vitest";
 // repo
 import {
-  GET_TESTID_WALKTHROUGH_CARD,
+  // GET_TESTID_WALKTHROUGH_CARD,
```

Comment:
are lines 5 and 8 also commented out with the TODO for HOUSNAV-191

Reply:
I just figured all of those would be caught when that file was revisited. The one there is more a note for me for the whole file.